scripts/local_testnet/ganache_test_node.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
+source ./vars.env
+
 ganache-cli \
 	--defaultBalanceEther 1000000000 \
 	--gasLimit 1000000000 \
 	--accounts 10 \
-	--mnemonic "vast thought differ pull jewel broom cook wrist tribe word before omit" \
+	--mnemonic $ETH1_NETWORK_MNEMONIC \
```

Comment:
We may want to enclose this bash variable in quotes, as in: `--mnemonic "$ETH1_NETWORK_MNEMONIC" \`, to get all the words of the mnemonic as opposed to only the first one.

Reply:
Thanks a lot, was wondering what went wrong with eth1.